pkg/trimaran/loadvariationriskbalancing/collector.go

```diff
@@ -66,10 +67,15 @@ func newCollector(obj runtime.Object) (*Collector, error) {
 	if args.WatcherAddress != "" {
 		client, _ = loadwatcherapi.NewServiceClient(args.WatcherAddress)
 	} else {
+		insecureSkipVerify := true
+		if args.MetricProvider.InsecureSkipVerify != "" {
```

Comment:
If you update defaults this logic can be handled there

Reply:
+1